runhouse/rns/defaults.py

```diff
@@ -31,7 +31,7 @@ class Defaults:
         "default_autostop": -1,
         "use_spot": False,
         "use_local_configs": True,
-        "disable_data_collection": False,
```

Comment:
we should still keep the data collection param, that's a separate type of data collection (ex: sentry)